### DISCUSSION TITLE
` Topic not found: sdf-cmd-reply `

### DISCUSSION THREAD

**Discussion by ArnavK-09** [fluvio-community/fluvio#4145] [Q&A] [answered]
randomly errors pops out in dataflow inline, unable to fix it tried restarting cluster, deleting etc. didn't worked :(

**Comment by sehz** [comment]
is way to reproduce it?  please provide `sdf version`

**Reply by ArnavK-09** [reply]
![image](https://github.com/user-attachments/assets/93516ad4-f776-4a7c-9f7b-bf5f1da15ede)

**Reply by sehz** [reply, marked as answer]
have you tried `sdf run --ephemeral`?

**Reply by ArnavK-09** [reply]
Worked 👍

**Reply by sehz** [reply]
Great.  We are going to reduce confusion in beta2 release

**Reply by ArnavK-09** [reply]
I'll wait to try it, thanks
